pkgs/development/interpreters/emilua/default.nix

```diff
@@ -18,6 +17,8 @@
 , libcap
 , liburing
 , openssl
+, cereal
+, cmake
```

Comment:
Why cmake and meson at the same time?

Reply:
The new version uses cereal, and pkg-config does not recognize it, only cmake does, which is used in the project for such cases.

Reply:
Then use `dontUseCmakeConfigure = true;`.

Reply:
Ok